File: tacas2020-benchmarks/vaphor/condn.java
class vapexp{
	static int i,N;
	static int a[];

static void main()
{
	N=Support.random();
	i=0;
	while(i<N)
        {
		if(a[i] < N)
		{
			a[i] = a[i];
		} else {
			a[i] = N;
		}
		i = i + 1;
        }

	i=0;
	while(i<N)
        {
		assert(a[i] <= N);
		i = i + 1;
	}
}
}

File: tacas2020-benchmarks/booster/condn.c
#define N

int main()
{
	int i;
	int a[N];

	for(i=0; i<N; i++)
	{
		if(a[i] < N) {
			a[i] = a[i];
		} else {
			a[i] = N;
		}
	}

	for(i=0; i<N; i++)
	{
		assert(a[i] <= N);
	}
	return 1;
}
